contracts/periphery/VotePreviewer.sol

```diff
@@ -0,0 +1,65 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity ^0.8.17;
+
+import { IERC20 } from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import { FixedPointMathLib } from "solmate/src/utils/FixedPointMathLib.sol";
+import { EscrowedEXA } from "./EscrowedEXA.sol";
+
+/// @title Vote Previewer
+/// @notice Contract to be consumed by voting strategies.
+contract VotePreviewer {
+  using FixedPointMathLib for uint256;
+
+  /// @custom:oz-upgrades-unsafe-allow state-variable-immutable
+  address public immutable exa;
+  /// @custom:oz-upgrades-unsafe-allow state-variable-immutable
+  IPool public immutable pool;
+  /// @custom:oz-upgrades-unsafe-allow state-variable-immutable
+  IERC20 public immutable gauge;
+  /// @custom:oz-upgrades-unsafe-allow state-variable-immutable
+  IExtraLending public immutable extraLending;
+  /// @custom:oz-upgrades-unsafe-allow state-variable-immutable
+  uint256 public immutable extraReserveId;
+
+  /// @custom:oz-upgrades-unsafe-allow constructor
+  constructor(address exa_, IPool pool_, IERC20 gauge_, IExtraLending extraLending_, uint256 extraReserveId_) {
+    exa = exa_;
+    pool = pool_;
+    gauge = gauge_;
+    extraLending = extraLending_;
+    extraReserveId = extraReserveId_;
+  }
+
+  function externalVotes(address account) external view returns (uint256 votes) {
+    uint256 liquidity = pool.balanceOf(account) + gauge.balanceOf(account);
+    votes += liquidity.mulDivDown(exa == pool.token0() ? pool.reserve0() : pool.reserve1(), pool.totalSupply());
```

Comment:
how does `pool.token0()` evaluate as a bool for the ternary statement? I understand that `token0` and `token1` are the 2 tokens in VELO pool, but how is it falsy?